pems-server/src/main/java/org/maroubra/pemsserver/monitoring/utsapi/WebApiRequest.java

```diff
@@ -0,0 +1,18 @@
+package org.maroubra.pemsserver.monitoring.utsapi;
+
+import com.google.gson.Gson;
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Path;
+import retrofit2.http.QueryMap;
+
+import java.util.List;
+import java.util.Map;
+
+public interface WebApiRequest {
+
+
+    @GET("json/")
```

Comment:
I noticed that there is also a `hc` endpoint, which provides data in a similar json format, but uses UTC dates (epoch/unix time), and numbers (doubles where necessary). Maybe it might be worth using that endpoint too?

Reply:
Added HC endpoint.